packages/shorebird_code_push_protocol/lib/src/messages/get_usage/get_usage_response.dart

```diff
@@ -40,6 +41,10 @@ class GetUsageResponse {
   /// monthly cost and any overages.
   @MoneyConverter()
   final Money currentPeriodCost;
+
+  /// Whether the subscription will be canceled
+  /// at the end of the current billing period.
+  final bool cancelAtPeriodEnd;
```

Comment:
I probably would have flipped the bool to be "auto renews" rather than "cancels after period", but lgtm.

Reply:
I believe this is to keep our variable naming in line with Stripe's. See https://stripe.com/docs/billing/subscriptions/cancel#cancel-at-the-end-of-the-current-billing-cycle.